src/app/GithubLogo.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import React from "react";
+
+type Props = {
+  className: string,
+  altText: string,
+};
+
+export default function GithubLogo(props: Props) {
+  return (
+    <svg
+      aria-labelledby="ghLogoTitle"
```

Comment:
Question from an SVG-noob: is it possible/okay to instead write

```jsx
<svg aria-label={props.altText} …>
  <title>{props.altText}</title>
  {/* … */}
</svg>
```

? I’m generally quite reluctant to use `id`s unless absolutely
necessary, because `id`s create global state on the page and thus break
encapsulation.

For instance, as the component is currently written, I can render

```jsx
<div>
  <GithubLogo className="" altText="Title the first" />
  <GithubLogo className="" altText="Title the second" />
</div>
```

and each SVG will have the correct _hover_ text, but both SVGs appear as
“Title the first” to screenreaders (according to tota11y, at least).

Reply:
I was basing it off of this guide: https://css-tricks.com/accessible-svgs/
But after further research, looks like `aria-label` should be fine.